components/AgentForm.tsx

```diff
@@ -768,7 +802,11 @@ export default function BaseForm(props: Props) {
           {'Save'}
         </Button>
       </form>
-
+      <Divider sx={{ my: 4 }} />
+      <RateLimitForm
+          onSubmit={handleSubmitRateLimit}
+          rateLimit={agentConfig.rateLimit}
```

Comment:
🚨 Error when trying to create a new Agent
Have you tested this feature?

Reply:
got the problem, which has to do with the the RateLimitForm, the creation works fine, but the save button on the rateLimitForm hit an update endpoint, I have fixed that